nixie/unittests/test_render_diagram.py

```
@@ -8,23 +8,18 @@
 
-from nixie.cli import _run_mermaid_cli, get_mmdc_cmd
+from nixie.cli import _render_diagram, get_mmdc_cmd
 
 
 @pytest.mark.asyncio
-async def test_run_mermaid_cli_invokes_command(
-    monkeypatch: pytest.MonkeyPatch,
-    tmp_path: Path,
-    caplog: pytest.LogCaptureFixture,
+async def test_render_diagram_writes_file_and_logs(
+    monkeypatch: pytest.MonkeyPatch, tmp_path: Path, caplog: pytest.LogCaptureFixture
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)